components/Editor/EditorControls.tsx

```diff
@@ -96,6 +129,16 @@ const EditorControls = ({
           'text-red-500': !areProgramArgumentsValid,
         })}
       />
+      <Select
+        isSearchable={false}
+        classNamePrefix="select"
+        menuPlacement="auto"
+        value={exampleNameValue}
+        options={examplesOptionos}
+        instanceId="exampleSelect"
```

Comment:
when my PR is merged you can use the React 18 useId() hook instead of hard-coded one

https://github.com/walnuthq/cairovm.codes/pull/105/files#diff-d7703e6081738cd218c05e03cfc118115be2b7200de2f7f55ec1f2f25bd33df9R40